osquery/events/linux/inotify.cpp

```diff
@@ -212,8 +212,15 @@ INotifyEventContextRef INotifyEventPublisher::createEventContextFrom(
   // Get the pathname the watch fired on.
   {
     WriteLock lock(path_mutex_);
-    ec->path = descriptor_paths_.at(event->wd);
+    if (descriptor_paths_.find(event->wd) == descriptor_paths_.end()) {
+      ec->action = "";
```

Comment:
Replaced the redundant setting of values with a comment that explicitly states the intent to return a blank context.